Tools/jit/_targets.py

```diff
@@ -176,41 +180,44 @@ async def _build_stencils(self) -> dict[str, _stencils.StencilGroup]:
             )
         return stencil_groups
 
-    def build(
-        self, out: pathlib.Path, *, comment: str = "", force: bool = False
-    ) -> None:
+    def build(self, out: pathlib.Path, *, force: bool = False) -> None:
         """Build jit_stencils.h in the given directory."""
         if not self.stable:
             warning = f"JIT support for {self.triple} is still experimental!"
             request = "Please report any issues you encounter.".center(len(warning))
             outline = "=" * len(warning)
             print("\n".join(["", outline, warning, request, outline, ""]))
-        digest = f"// {self._compute_digest(out)}\n"
+        digest = f"{self._compute_digest(out)}\n"
         jit_stencils = out / "jit_stencils.h"
-        if (
-            not force
-            and jit_stencils.exists()
-            and jit_stencils.read_text().startswith(digest)
-        ):
-            return
+        jit_stencils_digest = out / "jit_stencils.h.digest"
+
+        if not force and jit_stencils_digest.exists() and jit_stencils.exists():
+            if jit_stencils_digest.read_text() == digest:
+                return
+
+        # We need to ignore the version number for Darwin targets
```

Comment:
note to self to move this to build.py to hardcode the name of the stencil as a class attr and use that